lib/fs/JsongoFSCollection.ts

```diff
@@ -12,7 +12,7 @@ export class JsongoFSCollection extends JsongoCollection<JsongoFSDB> {
     try {
       const jsonBuf = this._fs().readFileSync(this._filePath(), "utf-8");
       this._docs = JSON.parse(jsonBuf);
-    } catch (ex) {
+    } catch (ex: any) {
```

Comment:
Ideally, this should include `if (ex instanceof NodeJS.ErrnoException && ex.code === "ENOENT")`. However, this expression raises `Cannot use namespace 'NodeJS' as a value`.

Also, `import { ErrnoException } from '@types/node'` doesn't work either - `File @types/node/index.d.ts' is not a module.`